segregation/spatial/spatial_indexes.py

```diff
@@ -10,7 +10,7 @@
 import warnings
 import libpysal
 
-from libpysal.weights import Queen, Kernel, lag_spatial
+from libpysal.weights import W, Queen, Kernel, lag_spatial
```

Comment:
this `W` isn't used

Reply:
Ok, indeed! I just merged that, but will address in another PR!

Reply:
wouldn't worry too much about it